wgpu-core/src/device/mod.rs

```diff
@@ -3013,12 +3013,12 @@ impl<G: GlobalIdentityHandlerFactory> Global<G> {
             .map_err(|_| instance::IsSurfaceSupportedError::InvalidSurface)?;
         Ok(adapter.is_surface_supported(surface))
     }
-    pub fn surface_get_preferred_format<A: HalApi>(
+    pub fn surface_get_supported_formats<A: HalApi>(
         &self,
         surface_id: id::SurfaceId,
         adapter_id: id::AdapterId,
-    ) -> Result<TextureFormat, instance::GetSurfacePreferredFormatError> {
-        profiling::scope!("surface_get_preferred_format");
+    ) -> Result<Vec<TextureFormat>, instance::GetSurfacePreferredFormatError> {
+        profiling::scope!("surface_get_supported_formats");
```

Comment:
```suggestion
        profiling::scope!("surface::get_supported_formats");
```

Reply:
Are you sure? The function itself is `surface_get_supported_formats`

Reply:
Yeah, we should be naming the functions the wgpu (not wgpu-core) name